### PR TITLE
New Design to search bar

### PR DESCRIPTION
**Pull request by JuanAlmeida12**
#### What is the purpose of this pull request?

Product count will be display below filter and order buttons.

#### What problem is this solving?

The new Design requires the product count will be displayed below buttons.

#### How should this be manually tested?
[Access this Workspace](https://newsearchresult--storecomponents.myvtex.com/shoes)

#### Screenshots or example usage

newsearchresult--storecomponents myvtex com_shoes pixel 2

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [x] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
```
Comment (brunojdo) [inline CHANGELOG.md]: When we have only one product remove the `s`

File: react/global.css [hunk 2]
```diff
@@ -213,3 +213,7 @@ body.vtex-filter-popup-open {
     grid-template-columns: repeat(2, 1fr);
   }
 }
+
+.vtex-dropdown__mobile {
+  margin-top: -2px;
```
Comment (ArthurSampaio) [inline react/global.css]: isit really necessary use a negative margin? I think that could be a problem if the parent component had any change 🤔

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+- Product count will be displayed below filter and order buttons
```
Comment (matheusps) [inline CHANGELOG.md]: Add '.' at the end of the sentence.

File: react/components/SearchResult.js [hunk 4]
```diff
@@ -155,13 +156,11 @@ class SearchResult extends Component {
     return (
       <LoadingOverlay loading={showLoading && showLoadingAsOverlay}>
         <div className="vtex-search-result vtex-p pv5 ph3">
+          {mobile && (<div className="vtex-search-result__divider bt bw1 b--muted-4 w-100"></div>)}
```
Comment (ArthurSampaio) [inline react/components/SearchResult.js]: Remove the `bw1` to improve the design cohesion

Comment (JuanAlmeida12) [reply react/components/SearchResult.js]: This line has been removed.

**Pull request merged**